content/docs/error-boundaries.md

```diff
@@ -129,6 +136,34 @@ However, React components are declarative and specify *what* should be rendered:
 
 Error boundaries preserve the declarative nature of React, and behave as you would expect. For example, even if an error occurs in a `componentDidUpdate` hook caused by a `setState` somewhere deep in the tree, it will still correctly propagate to the closest error boundary.
 
+However, a typical event handler inside a React component could leverage `try` / `catch` to deal with errors occured during event handling.
```

Comment:
How about a slight wording tweak?

> However, an event handler inside of a React component could leverage `try` / `catch` to deal with errors that occur during event handling.

Reply:
I agree with this. The word `typical` is somewhat confusing.